runelite-client/src/main/java/net/runelite/client/plugins/fps/FpsIndicatorLocation.java

```diff
@@ -0,0 +1,21 @@
+package net.runelite.client.plugins.fps;
```

Comment:
Why is this needed?

Reply:
This is like purely debug information, i dont think we need fancy toggles

Reply:
It covers up the stretch mode X button thing in that mode

Reply:
Well, then we can just detect that and move both overlays down

Reply:
Yeah I added this because people complained about it covering the log out button in discord, not sure if an issue was ever posted though

Reply:
I'm really not sure how I feel about that config option anyway. It blocks the left click option text so it's kind of out of place. I'll probably just update this to move a bit to the left when the logout button is in the corner